### ISSUE TITLE
1. Request long run of barcode numbers from Specify team for NHMA, create barcode labels, print, cut, and send

### ISSUE THREAD

**Issue by chelseagraham** [NHMDenmark/Pinned-Insects-workstation#77]
- [x] Chelsea to double check with Specify team to ensure these numbers are still available and request 15.000 more numbers.
- [x] Matilde to create labels for the numbers
- [ ] Technical team to print labels
- [ ] Technical team to send printed labels to NHMA

Status:
@chelseagraham previously mailed labels 500,000 - 501,619 in June.
@Gomismis previously mailed labels 501,620 - 503,239 in September.
@Gomismis delivered labels 503,240 - 508,099 in October.
@chelseagraham mailed labels 508,100 - 509,719 on 15 April 2025.

**Comment by chelseagraham** [comment]
After some discussion about what numbers were actually available, Hans replied with a desired number run. 
Fedor reserved 500000 to 525000. 

[RE: NHMA Entomology #s.pdf](https://github.com/user-attachments/files/25259996/RE.NHMA.Entomology.s.pdf)

**Comment by Gomismis** [comment]
I generated the labels and uploaded them here as a PDF-file. 

[NHMAlabels_100000-110000.PDF](https://github.com/NHMDenmark/Pinned-Insects-workstation/files/14601590/NHMAlabels_100000-110000.PDF)

**Comment by chelseagraham** [comment]
This is contingent on resolution on label design /redesign https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/40

**Comment by chelseagraham** [comment]
As of today, 10 April, we have not received any suggested redesign from NHMA (discussed at meeting on 18 March 2024).

**Comment by chelseagraham** [comment]
The new label design has been agreed upon. https://github.com/NHMDenmark/DaSSCo-Image-Refinery/issues/280
@Gomismis would you please see if you can use the code to generate the labels in the new design?

I have ordered the necessary cartridge from Canon for our printer. If we do not receive it by Tuesday, let's plan to print these on a colleague's printer and get them sent out to NHMA by mid week next week.

**Comment by Gomismis** [comment]
I have created the new labels 
[NHMA_500000-520000_labels.PDF](https://github.com/user-attachments/files/15821712/NHMA_500000-520000_labels.PDF)

They will be printet as soon as possible

**Comment by Gomismis** [comment]
I have created smaller PDF's for printing. I have printed 3/25 PDF's (1619 labels). Unfortunately our printer is making troubles but I'm trying to fix it ASAP. I will continuing printing when the printer is usable again.

**Comment by chelseagraham** [comment]
On Monday, 24 June, Chelsea sent out the printed labels: 500,000 - 501,619 to NHMA (Naturhistorisk Museum, Hans Viborg Kristensen, Wilhelm Meyers Alle 10, 8000 Aarhus C) in materials fro the Depot on floor 1 of ZM. I wrote NHMA's address of a label and also affixed a form to inform Pakkecentralen of the information pertaining to the envelope. 

[Template - Proforma Invoice_NHMA LABELS.pdf](https://github.com/user-attachments/files/15969906/Template.-.Proforma.Invoice_NHMA.LABELS.pdf)

N:/SNM-DigitalCollections/DaSSCo/Workflows and workstations/All workstations documents/Template - Mailslip_NHMA LABELS.doc

**Comment by chelseagraham** [comment]
We are at a standstill because of the issues with the printer. Canon Pro-1000, but the decision has been made to purchase an older model printer which we can control to only use the carbon pigmented ink cartridges and Anders and Matilde will be in discussion about how to purchase a model that is used by an SNM colleague and for which there is still ink. So, hopefully soon we can resume printing labels to send out to NHMA before their Digitizer starts 12 August.

**Comment by chelseagraham** [comment]
NHMA requested a revamp of their labels. Thomas Alcher worked on this and is waiting for THomas Simonsen's approval. 
https://github.com/NHMDenmark/DaSSCo-Image-Refinery/issues/280#issuecomment-2238679767

**Comment by chelseagraham** [comment]
The label redesign is still being confirmed (now here https://github.com/NHMDenmark/DaSSCo-Image-Refinery/issues/307)

**Comment by chelseagraham** [comment]
Wrote to Hans & Charlotte (copying Matilde) to inquire about how their stock of labels is doing (no. 500,000 - 501,619 sent on 24 June) 
and to confirm that they are planning on continuing with this size label for pyralidae and other lepidoptera 
so that they can be in touch with Matilde regarding their use and she can print and send more labels out to NHMA before they run out.

**Comment by Gomismis** [comment]
I have printed PDF 3 and 4  (additional 1619 numbers) and mailed them to Århus.

**Comment by Gomismis** [comment]
Printed PDFs: 
- [x] [NHMA_500000-520000_labels-1.pdf](https://github.com/user-attachments/files/17058059/NHMA_500000-520000_labels-1.pdf)

- [x] [NHMA_500000-520000_labels-2.pdf](https://github.com/user-attachments/files/17058061/NHMA_500000-520000_labels-2.pdf)

- [x] [NHMA_500000-520000_labels-3.pdf](https://github.com/user-attachments/files/17058058/NHMA_500000-520000_labels-3.pdf)

- [x] [NHMA_500000-520000_labels-4.pdf](https://github.com/user-attachments/files/17058060/NHMA_500000-520000_labels-4.pdf)

- [x] [NHMA_500000-520000_labels-5.pdf](https://github.com/user-attachments/files/17058011/NHMA_500000-520000_labels-5.pdf)

- [x] [NHMA_500000-520000_labels-6.pdf](https://github.com/user-attachments/files/17058024/NHMA_500000-520000_labels-6.pdf)

- [x] [NHMA_500000-520000_labels-7.pdf](https://github.com/user-attachments/files/17058020/NHMA_500000-520000_labels-7.pdf)

- [x] [NHMA_500000-520000_labels-8.pdf](https://github.com/user-attachments/files/17058017/NHMA_500000-520000_labels-8.pdf)

- [x] [NHMA_500000-520000_labels-9.pdf](https://github.com/user-attachments/files/17058012/NHMA_500000-520000_labels-9.pdf)

- [x] [NHMA_500000-520000_labels-10.pdf](https://github.com/user-attachments/files/17058016/NHMA_500000-520000_labels-10.pdf)

- [x] [NHMA_500000-520000_labels-11.pdf](https://github.com/user-attachments/files/17058022/NHMA_500000-520000_labels-11.pdf)

- [x] [NHMA_500000-520000_labels-12.pdf](https://github.com/user-attachments/files/17058015/NHMA_500000-520000_labels-12.pdf)

- [x] [NHMA_500000-520000_labels-13.pdf](https://github.com/user-attachments/files/17058013/NHMA_500000-520000_labels-13.pdf)

- [x] [NHMA_500000-520000_labels-14.pdf](https://github.com/user-attachments/files/17058025/NHMA_500000-520000_labels-14.pdf)

- [x] [NHMA_500000-520000_labels-15.pdf](https://github.com/user-attachments/files/17058023/NHMA_500000-520000_labels-15.pdf)

- [x] [NHMA_500000-520000_labels-16.pdf](https://github.com/user-attachments/files/17058026/NHMA_500000-520000_labels-16.pdf)

- [x] [NHMA_500000-520000_labels-17.pdf](https://github.com/user-attachments/files/17058014/NHMA_500000-520000_labels-17.pdf)

- [x] [NHMA_500000-520000_labels-18.pdf](https://github.com/user-attachments/files/17058009/NHMA_500000-520000_labels-18.pdf)

- [x] [NHMA_500000-520000_labels-19.pdf](https://github.com/user-attachments/files/17058027/NHMA_500000-520000_labels-19.pdf)

- [x] [NHMA_500000-520000_labels-20.pdf](https://github.com/user-attachments/files/17058019/NHMA_500000-520000_labels-20.pdf)

- [x] [NHMA_500000-520000_labels-21.pdf](https://github.com/user-attachments/files/17058010/NHMA_500000-520000_labels-21.pdf)

- [x] [NHMA_500000-520000_labels-22.pdf](https://github.com/user-attachments/files/17058018/NHMA_500000-520000_labels-22.pdf)

- [x] [NHMA_500000-520000_labels-23.pdf](https://github.com/user-attachments/files/17058029/NHMA_500000-520000_labels-23.pdf)

- [x] [NHMA_500000-520000_labels-24.pdf](https://github.com/user-attachments/files/17058028/NHMA_500000-520000_labels-24.pdf)

- [x] [NHMA_500000-520000_labels-25.pdf](https://github.com/user-attachments/files/17058021/NHMA_500000-520000_labels-25.pdf)

Last 5000 Big labels

- [x] [run520001-525000_Part1.pdf](https://github.com/user-attachments/files/25517973/run520001-525000_Part1.pdf)

- [x] [run520001-525000_Part2.pdf](https://github.com/user-attachments/files/25517974/run520001-525000_Part2.pdf)

- [x] [run520001-525000_Part3.pdf](https://github.com/user-attachments/files/25517972/run520001-525000_Part3.pdf)

- [x] [run520001-525000_Part4.pdf](https://github.com/user-attachments/files/25517976/run520001-525000_Part4.pdf)

- [x] [run520001-525000_Part5.pdf](https://github.com/user-attachments/files/25517975/run520001-525000_Part5.pdf)

- [x] [run520001-525000_Part6.pdf](https://github.com/user-attachments/files/25517971/run520001-525000_Part6.pdf)

**Comment by chelseagraham** [comment]
@Gomismis which label run did you deliver to NHMA last week?

**Comment by Gomismis** [comment]
the ones that is crossed off as printed in this ticket (PDF 5-10)

**Comment by chelseagraham** [comment]
All of the PDFs that have been previously sent are crossed off in this ticket, not just the ones most recently delivered.

I'd like to keep track on this ticket to the actual numbers of the labels as well, as [here](https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/77#issuecomment-2357837086), if possible. 

As an update:
@chelseagraham previously mailed labels 500,000 - 501,619 in June. 
@Gomismis previously mailed labels 501,620 - 503,239 in September.
@Gomismis delivered labels 503,240 - 508,099 in October.

**Comment by Gomismis** [comment]
sorry it was a typo, I have edited now :) (PDF 5-10)

**Comment by chelseagraham** [comment]
We have been printing these labels, but seeing as it is a big range (edit: 25,000 numbers) and NHMA has requested a smaller label be generated for future work (see https://github.com/NHMDenmark/DaSSCo-Image-Refinery/issues/307), I inquired whether they could anticipate how many more labels they needed of this original size before we start using some of these numbers for the new label size, or if they plan to use the rest of this number series for lepidoptera and start a new number series for the new labels.

**Comment by chelseagraham** [comment]
In the Pinned Insect Catch-Up Meeting in February
https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/120#event-16496357080

Thomas Simonsen said that they would optimally like to have the new smaller size labels in a couple of weeks. I spoke to Allison to let her know about their expectations and inquire whether it could be possible. She said that she would try and moved it up in her priorities. 

Thomas also said that once they get the new smaller size labels, they would like to cease using the original Lepidoptera labels. 

@Gomismis you can stop printing the labels that we have generated for their number run.

@RebekkaML will you please check where NHMA is at with their labels. 

We checked in September and they had 1000 left, so we sent them 1619 labels in September and then Matilde brought 4859 labels in October. Probably, NHMA won't need more labels in the original Lepidoptera size. 

When we do have the new size label ready, we will have to check in with them at what number they are at and then make sure that we either print an overlap and ask them to start from where they leave off with the big labels, or agree where they will start with the new size and make the new labels from that number.

**Comment by chelseagraham** [comment]
> In the Pinned Insect Catch-Up Meeting in February [#120 (comment)](https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/120#event-16496357080)
> 
> Thomas Simonsen said that they would optimally like to have the new smaller size labels in a couple of weeks. I spoke to Allison to let her know about their expectations and inquire whether it could be possible. She said that she would try and moved it up in her priorities.
> 
> Thomas also said that once they get the new smaller size labels, they would like to cease using the original Lepidoptera labels.
> 
> [@Gomismis](https://github.com/Gomismis) you can stop printing the labels that we have generated for their number run.
> 
> [@RebekkaML](https://github.com/RebekkaML) will you please check where NHMA is at with their labels.
> 
> We checked in September and they had 1000 left, so we sent them 1619 labels in September and then Matilde brought 4859 labels in October. Probably, NHMA won't need more labels in the original Lepidoptera size.
> 
> When we do have the new size label ready, we will have to check in with them at what number they are at and then make sure that we either print an overlap and ask them to start from where they leave off with the big labels, or agree where they will start with the new size and make the new labels from that number.



Thomas checked back in after the meeting notes went out to say that they had discussed it at NHMA and he had misspoken during the meeting. NHMA would like to continue using the original labels for some specimens, but would also like to have the new labels created. 

Today NHMA got back to us with the decision that they would like to start a new number series for the new labels.

@Gomismis , we can continue printing the original labels regularly to keep the printer healthy. 
We are not close to it yet, but some day would you please generate the last 5,000 labels and stick the PDFs in the comment with the rest?
We can mail them their labels periodically.

**Comment by chelseagraham** [comment]
On Tuesday, 15 April, Chelsea sent out the printed labels: 508,100 - 509,719 to NHMA (Naturhistorisk Museum, Hans Viborg Kristensen, Wilhelm Meyers Alle 10, 8000 Aarhus C) in materials fro the Depot on floor 1 of ZM. I wrote NHMA's address of a label and also affixed a form to inform Pakkecentralen of the information pertaining to the envelope.

[Template - Proforma Invoice_NHMA LABELS.pdf](https://github.com/user-attachments/files/15969906/Template.-.Proforma.Invoice_NHMA.LABELS.pdf)

N:/SNM-DigitalCollections/DaSSCo/Workflows and workstations/All workstations documents/Template - Mailslip_NHMA LABELS.doc

**Comment by chelseagraham** [comment]
We should continue printing some labels for NHMA every week or two to keep the printer rolling. We can collect them in a lower tier of our gray paper filer in the wooden shelving in 249.

**Comment by chelseagraham** [comment]
@RebekkaML, @Gomismis, @annejodobel will you please print some PDFs from this run periodically this summer. I've labeled the paper filer with the institutions and GitHub tickets, so you can easily see into which slot to put your prints :) 
The slot above the label NHMA 77 is the correct one for this run :)

**Comment by chelseagraham** [comment]
I am checking in with Charlotte about the stock left and their needs for more barcodes before I am out on graviditetsorlov. 

On 15 April 2025, I sent labels 508,100 - 509,719 (PDFs 11-12).

I can see that since then, PDFs 13-17 have been printed. 

Now that we are set up, post-move, in the new room at ZM, would it be possible for everyone to print some labels when there and place in the correct tray for this GitHub ticket? I'd love to be able to send them the rest before I go on leave. The last days I plan to spend at ZM are 3 and 10 March, so it would be great if we could aim to have all PDFs printed before 3 March and 10 March as a back-up.

**Comment by chelseagraham** [comment]
I also just double checked and the PDF file names are not a typo, they end with no. 520.000. We need labels for the entire run - up to 525.000. @Gomismis would you please generate these and append them to this ticket?

**Comment by chelseagraham** [comment]
Charlotte reports that of this original run, she has 10 sheets left + 11 from a former batch.

I would still like to send the rest of the run, barcode label numbers 509.720-525.000 before my leave and hopefully on 3 March. 

@Gomismis will you please generate the missing barcode labels and link them in this ticket.
@Gomismis @RebekkaML @annejodobel will you please coordinate to print the rest of the series and organize them in the paper tray under NHMA #77 in the Digitization room at ZM?

**Comment by RebekkaML** [comment]
These are all printed now and can be sent to Aarhus.

**Comment by chelseagraham** [comment]
Rock on! Thank you, Rebekka! Great to have this complete. I think we should wait to mail these together with those from https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/153 though

**Comment by chelseagraham** [comment]
On Tuesday, 3 March, Chelsea sent out the printed labels: 509,720 - 525,000 to NHMA (Naturhistorisk Museum, Hans Viborg Kristensen, Wilhelm Meyers Alle 10, 8000 Aarhus C) in materials from the Depot on floor 1 of ZM. I wrote NHMA's address on a label and also affixed a form to inform Pakkecentralen of the information pertaining to the envelope.

[Template - Proforma Invoice_NHMA LABELS.pdf](https://github.com/user-attachments/files/15969906/Template.-.Proforma.Invoice_NHMA.LABELS.pdf)

N:/SNM-DigitalCollections/DaSSCo/Workflows and workstations/All workstations documents/Template - Mailslip_NHMA LABELS.doc

--------------

This completes the series. There are no more labels that will need to be printed and sent to NHMA in this size before the conclusion of DaSSCo. I am closing the ticket.

----------

Just want to make @kimstp & @bhsi-snm aware that we have satisfied our label provision commitment. 
DaSSCo can now package up the code for this macro moth label size and deliver it to NHMA for their future label generation.
